src/base64.c

```diff
@@ -0,0 +1,273 @@
+/*
+ * See Licensing and Copyright notice in naev.h
+ */
+
+/*
+ * compliant with rfc3548
+ */
+
+#include "base64.h"
+
+#include <stdlib.h>
+#include <stdint.h>
+
+#include "nstring.h"
+
+
+/* encode table - base 64 alphabet as defined by the rfc */
+static const char cb64[64]="ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/";
+/* generate decode table at compile time */
+#define B64(_)                   \
```

Comment:
The fact that `_` is being used for the argument on this is my only criticism. Not causing a problem I guess, it just makes me feel possibly unjustifiably nervous. :wink:

🕵️